File: src/main/java/ua/com/alevel/ma12/java/entity/Car.java
package ua.com.alevel.ma12.java.entity;

public class Car {

    int wheels = 4;
    int door =4;
    int windows =6;
    String name;
    String model;
    double price;
    int speed;

    public Car(){

    }

    public Car(String name, String model, double price, int speed){
        this.name = name;
        this.model = model;
        this.price = price;
        this.speed = speed;
    }

    @Override
    public String toString() {
        return "Car{" +
                "wheels=" + wheels +
                ", door=" + door +
                ", windows=" + windows +
                ", name='" + name + '\'' +
                ", mark='" + model + '\'' +
                ", price=" + price +
                ", speed=" + speed +
                '}';
    }
}

File: src/main/java/ua/com/alevel/ma12/kotlin/Car.kt
package ua.com.alevel.ma12.kotlin

class Car {

    var wheels:Int = 4;
    var windows:Int = 6;
    var speed:Int = 0;
    var price:Double = 0.0;
    var name:String = "";
    var mark:String = "";

    constructor(speed: Int, price: Double, name: String, mark: String) {
        this.speed = speed
        this.price = price
        this.name = name
        this.mark = mark
    }

    override fun toString(): String {
        return "Car(wheels=$wheels, windows=$windows, speed=$speed, price=$price, name='$name', mark='$mark')"
    }


}
